src/transform/DimImagesTransform.js

```diff
@@ -0,0 +1,25 @@
+import './DimImagesTransform.css'
+const CLASS = 'pagelib-dim-images'
+
+/**
+ * @param {!Window} window
+ * @param {!boolean} shouldDim
+ * @return {void}
+ */
+const dimImages = (window, shouldDim) => {
```

Comment:
I'm a little confused by this naming choice. You mentioned removing redundancy from the naming as a goal so why should we have a `DimImagesTransform.dimImages()` instead of just a `DimImagesTransform.dim()`?